### DISCUSSION TITLE
How do i create a Branch using the terminal in VS Code?

### DISCUSSION THREAD

**Discussion by BorisPhilipp** [community/community#89927] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I've seen that it is possible to create a branch only using the terminal, how do i do it?

**Comment by joaonakano** [comment, marked as answer]
With the open terminal, type the command `git branch name_of_the_branch`.
To enter this new branch, you can also type `git checkout name_of_the_branch`.

Hope I helped!

**Reply by BlessingsMinga** [reply]
Thank you

**Reply by refacta** [reply]
For anyone interested, you can achieve the same result with only one command, by using `git checkout -b name_of_the_branch`. This combines the two separate commands into one - nice for those who typically always checkout a branch straight after creating it. Also means you're not kicking yourself when you realise you've been working on the wrong branch because you forgot to checkout.

**Comment by Mo-TawabDev** [comment]
for created = git checkout -b nameofthebranch
